cinema/models.py

```diff
@@ -35,12 +40,20 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def uploads_movies(instance, filename):
+    extension = os.path.splitext(filename)[1]
+    filename = f"{slugify(instance.title)}--{uuid.uuid4().hex}{extension}"
```

Comment:
The `uploads_movies` function is defined to handle file uploads with a specific naming convention, but it is not used in the `Movie` model's `image` field. Consider setting `upload_to=uploads_movies` in the `ImageField` to utilize this function.